resources/views/page-builder/blog_quicklook.blade.php

```diff
@@ -0,0 +1,26 @@
+@php($blogs = \Statamic\Statamic::tag('collection:blog')->fetch())
```

Comment:
Maybe we should apply the limit here so we're not querying all blogs first to limit it afterwards.